### DISCUSSION TITLE
Help with multi-user QC

### DISCUSSION THREAD

**Discussion by HaakonEngen** [HALFpipe/HALFpipe#396] [Q&A] [answered]
Hi there, 

First, thanks for HALFpipe; it's an absolute treasure. 

Second: We're stuck! 

So, we're on a RHEL HPC/cluster, which I've used successfully to run preproc on a bunch of subjects. Now, in order to remain sane, I'm trying to organise a little army of RAs to do QA. 

What we're running into is that the index.html in /reports doesn't update for me (or between them) as they go through subjects. I'm guessing this is a caching issue or something fun like that, but I can't really figure out how to resolve it. 

So... HELP!??!?

**Comment by HippocampusGirl** [comment, marked as answer]
Hi @HaakonEngen, 

When the ratings are made, they are only saved locally in the web browser of the specific user, meaning they are not shared.

Your students need to export their ratings, and then you can import them on your computer. 

![image](https://user-images.githubusercontent.com/789054/183670984-c4758bbf-29e1-4953-8ec1-2506e2e95314.png)

**Comment by HaakonEngen** [comment]
Aha! The version we're using doesn't have the import functionality. When was that added? Will it work with exports from a previous version?

Thank you for the clarification!

H. 

On 9 August 2022 14:11:50 UTC, Lea Waller ***@***.***> wrote:
>Hi @HaakonEngen, 
>
>When the ratings are made, they are only saved locally in the web browser of the specific user, meaning they are not shared.
>
>Your students need to export their ratings, and then you can import them on your computer. 
>
>![image](https://user-images.githubusercontent.com/789054/183670984-c4758bbf-29e1-4953-8ec1-2506e2e95314.png)
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/HALFpipe/HALFpipe/discussions/396#discussioncomment-3358488
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by HippocampusGirl** [comment]
To update to the newest version, just replace the `index.html` file in the reports folder with the most recent one. At time of writing, that is <https://github.com/HALFpipe/QualityCheck/releases/download/0.4.1/index.html>
